.github/workflows/integration_tests.yaml

```diff
@@ -0,0 +1,41 @@
+name: integration-tests 
+on:
+  workflow_call:
+    inputs:
+      lorax_tag:
+        required: true
+        type: string
+    secrets:
+      RUNPOD_API_KEY:
+        required: true
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install runpod
+        run: |
+          wget -qO- cli.runpod.net | sudo bash
+      - name: Config runpod cli 
+        run: |
+          runpodctl config --apiKey ${{ secrets.RUNPOD_API_KEY }}
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      
+      # Install the client 
+      - run: pip install -e clients/python
+      - name: create the pod and then delete the pod
+        run: |
+          bash tests/create-pod.sh ${{ inputs.lorax_tag }}
+      
+      - name: Run tests 
+        run: |
+          POD_ID=`cat pod_name.txt`
+          python3 tests/test.py $POD_ID
```

Comment:
Where is this coming from?

Reply:
I might make it more obvious, its coming from `create-pod.sh`

I now see that I didn't commit that folder...